integration_tests/integration/court-register.spec.ts

```diff
@@ -179,6 +180,28 @@ context('Court register', () => {
     CourtDetailsPage.verifyOnPage('Sheffield Magistrates Court').markAsOpenButton().click()
     CourtDetailsPage.verifyOnPage('Sheffield Magistrates Court').activatedConfirmationBlock().should('exist')
   })
+  describe('amending a court', () => {
+    beforeEach(() => {
+      IndexPage.verifyOnPage().courtRegisterLink().click()
+      AllCourtsPage.verifyOnPage().viewCourtLink('SHFMC').should('contain.text', 'Sheffield Magistrates Court').click()
+    })
+    it('should show summary of court with link to amend', () => {
+      const courtDetailsPage = CourtDetailsPage.verifyOnPage('Sheffield Magistrates Court')
+      courtDetailsPage.courtDetailsSection().should('contain.text', 'Sheffield Magistrates Court')
+      courtDetailsPage.courtDetailsSection().should('contain.text', 'Sheffield Magistrates Court - Yorkshire')
+      courtDetailsPage.amendCourtDetailsLink().should('contain.text', 'Change')
+    })
+    it('can navigate to amend court details page', () => {
+      const courtDetailsPage = CourtDetailsPage.verifyOnPage('Sheffield Magistrates Court')
+      courtDetailsPage.amendCourtDetailsLink().click()
+      const amendCourtDetailsPage = AmendCourtDetailsPage.verifyOnPage('SHFMC')
+
+      amendCourtDetailsPage.name().should('have.value', 'Sheffield Magistrates Court')
+      amendCourtDetailsPage.description().should('have.value', 'Sheffield Magistrates Court - Yorkshire')
+      amendCourtDetailsPage.type().should('have.value', 'MAG')
+      amendCourtDetailsPage.saveButton().should('contain.text', 'Save')
```

Comment:
seems a bit overkill to test the save button here since the `cy` selector is to grab the element with `Save` as the text anyway?

Reply:
Yes, you are right. In the next part I will be just clicking it